src/Pages/LandingPage/LandingPage.jsx

```diff
@@ -17,16 +20,33 @@ const LandingPage = () => {
     setShowSignup(false);
   };
 
+  const onThemeTogglerClick = () => {
+    setDarkTheme((preTheme) => !preTheme);
+  };
+  const themeIcon = darkTheme ? "fa fa-sun" : "fa fa-moon";
+  const navBarClass = darkTheme
+    ? "desktop-navigation-bar dark-mode"
+    : "desktop-navigation-bar";
+
   return (
     <div className="landing">
       <div className="landing-header">
-        <img src={logo} alt="logo" className="landing-logo" />
-        <div className="landing-login-section">
+        <img
+          src={darkTheme ? logoLight : logoDark}
+          alt="logo"
+          className="landing-logo"
+        />
```

Comment:
Suggestion: Dark Mode Functionality can be better , toggling the classes throughout the multiple files code. It can be done with CSS Toggles. Do look into it